coded_tools/agent_network_designer/persist_agent_network.py

```diff
@@ -16,24 +16,25 @@
 
 import logging
 from copy import deepcopy
+from os import environ
 from typing import Any
 
 from neuro_san.interfaces.coded_tool import CodedTool
 
 from coded_tools.agent_network_designer.agent_network_assembler import AgentNetworkAssembler
 from coded_tools.agent_network_designer.agent_network_persistor import AgentNetworkPersistor
 from coded_tools.agent_network_designer.agent_network_persistor_factory import AgentNetworkPersistorFactory
+from coded_tools.agent_network_editor.constants import AGENT_NETWORK_DEFINITION
+from coded_tools.agent_network_editor.constants import AGENT_NETWORK_NAME
 from coded_tools.agent_network_validator import AgentNetworkValidator
 
-# To use reservations, turn this boolean to False and
+# To use reservations, turn this environment variable to true and also
 # export AGENT_TEMPORARY_NETWORK_UPDATE_PERIOD_SECONDS=5
-WRITE_TO_FILE: bool = True
+WRITE_TO_FILE: bool = environ.get("AGENT_NETWORK_DESIGNER_USE_RESERVATIONS", "false") != "true"
```

Comment:
Allow the use of reservations to come by way of an environment variable.
This will allow MAUI no-file-system-mods usage to be configurable in the container.